02.calendar/calendar.js

```diff
@@ -0,0 +1,49 @@
+import minimist from "minimist";
+
+function calcFirstDate() {
+  const argv = minimist(process.argv.slice(2));
+  const date = new Date();
+
+  if (argv["y"] >= 1970 && argv["y"] <= 2100) {
```

Comment:
オブジェクトのプロパティにアクセスするのにブラケット記法を使う必要は今回はないと思います。そうしないとアクセスできないようなプロパティ名ではないからです。ドット記法を使ってください 🙏 また、数字の範囲条件は以下のように書いておいた方が数直線と対応していて見やすいかなーと思います。この 2 つは月の処理についても同様です。

```suggestion
  if (1970 <= argv["y"] && argv["y"] <= 2100) {
```

Reply:
修正したら、とても見やすくなりました！